BrainPortal/config/console_rc.rb

```diff
@@ -91,24 +91,25 @@ def bb_bash(*bb)
   return false if bourreau_list.blank?
 
   # Prevents AR logging while we work here
-  no_log()
+  no_log do
 
-  # Run command on each
-  bourreau_list.each do |b|
-    puts "================ #{b.name} ================"
-    comm = yield(b)
-    if ! comm.is_a?(String)
-      puts "Block returned no string: #{comm.inspect}"
-    else
-      if b.proxied_host.present?
-        comm = "ssh #{b.proxied_host.bash_escape} #{comm.bash_escape}"
+    # Run command on each
+    bourreau_list.each do |b|
+      puts "================ #{b.name} ================"
+      comm = yield(b)
+      if ! comm.is_a?(String)
+        puts "Block returned no string: #{comm.inspect}"
+      else
+        if b.proxied_host.present?
+          comm = "ssh #{b.proxied_host.bash_escape} #{comm.bash_escape}"
```

Comment:
Could make it easier to read if you had
comm = "ssh #{b.proxied_host.bash_escape} #{comm.bash_escape}"  if b.proxied_host.present?